### DISCUSSION TITLE
The nightly build PPA for Ubuntu, parallel to the stable?

### DISCUSSION THREAD

**Discussion by bernd-wechner** [FreeRDP/FreeRDP#9256] [Q&A] [answered]
I have xfreerdp stable installed. 

I'd like to try out the nightlies as per: https://ci.freerdp.com/job/freerdp-nightly-binaries/

But the question I have before I add that PPA is, can I do that without clobbering my stable install. That is, is the nightly a distinctly named app that can install beside the stable? Or is there a way to configure the install (with apt?) to install it as such, like `xfreeerdp-nightly` for example?

**Comment by akallabeth** [comment, marked as answer]
the nigthlies install to `/opt/freerdp-nightly`, so don´t worry about having your original stable packages replaced.
you need to start the nighly with full path though (or add `/opt/freerdp-nightly/bin` to your path (before your current one)
another option is using our flatpak from flathub (we have stable and `beta` builds)
